### PR TITLE
Update vipps-checkout-how-it-works-woocommerce.md

### PR DESCRIPTION
**Pull request by jorgensel**
Added general part

Added images for Checkout specific stuff

Added GIFs

### CODE REVIEW

File: vipps-checkout-how-it-works-woocommerce.md [hunk 1]
```diff
@@ -9,6 +9,57 @@ END_METADATA -->
 
 With [Vipps' official WooCommerce plugin](https://wordpress.org/plugins/woo-vipps/), it's easy to start accepting payments with Vipps, VISA or MasterCard through Vipps Checkout.
 
+[See how it improves your WooCommerc Checkout experience here](https://www.youtube.com/watch?v=86RSKuQ5GME)
```
Comment (rebekaburnett) [inline vipps-checkout-how-it-works-woocommerce.md:12]:
```suggestion
See how it improves your WooCommerce Checkout experience [here](https://www.youtube.com/watch?v=86RSKuQ5GME)
```

File: vipps-checkout-how-it-works-woocommerce.md [hunk 2]
```diff
@@ -9,6 +9,57 @@ END_METADATA -->
...
 If you have a valid set of Vipps API keys, which means if you have a *Vipps på nett* or *Vipps Checkout* agreement, you can start using Vipps Checkout.
 
-The installation process is described here
+If you don't have an agreement with Vipps, [click here to order](https://portal.vipps.no/register/vippscheckout).
+
+# Vipps for WooCommerce plugin installation
+
+The installation process is described [here](https://github.com/vippsas/vipps-woocommerce), but can be summarised as:
+
+1.  Install the plugin using WordPress’ [built-in installer](https://codex.wordpress.org/Managing_Plugins#Installing_Plugins). The plugin can also be installed manually by upload the plugin files to the /wp-content/plugins/ directory.
```
Comment (rebekaburnett) [inline vipps-checkout-how-it-works-woocommerce.md:22]:
```suggestion
1.  Install the plugin using the WordPress [built-in installer](https://codex.wordpress.org/Managing_Plugins#Installing_Plugins). The plugin can also be installed manually by uploading the plugin files to the `/wp-content/plugins/` directory.
```

File: vipps-checkout-how-it-works-woocommerce.md [hunk 3]
```diff
@@ -9,6 +9,57 @@ END_METADATA -->
...
+2. Activate the plugin through the 'Plugins' screen in WordPress.
+3. Go to the WooCommerce Settings page and choose Payment Gateways (Betalinger) and enable Vipps.
+4. Go the settings page for the Vipps plugin and enter your Vipps account keys. Your account keys are available in the [Vipps Developer Portal](https://portal.vipps.no)
```
Comment (rebekaburnett) [inline vipps-checkout-how-it-works-woocommerce.md:25]:
```suggestion
4. Go the settings page for the Vipps plugin and enter your Vipps account keys. Your account keys are available in the [Vipps Developer Portal](https://portal.vipps.no).
```

File: vipps-checkout-how-it-works-woocommerce.md [hunk 4]
```diff
@@ -9,6 +9,57 @@ END_METADATA -->
...
+
+Vipps Checkout specific features are accessed in the "Payments" settings in the admin panel.
+![image](https://user-images.githubusercontent.com/25223283/226337801-7561a625-4ad5-4a68-aa56-96a6c1bcaf68.png)
+
+To turn on Checkout, simply check the box at the top of the "Vipps Checkout" screen.
+![image](https://user-images.githubusercontent.com/25223283/226338565-250873b7-ff9d-449a-8b7e-ce7392441a2c.png)
+
+# Improve your shipping selection
+
+Standard WooCommrce shipping selection is not optimal, so we have integrated with the major Norwegian shipping providers, to remove friction from an important step in the sales process. This list currentlig includes
```
Comment (rebekaburnett) [inline vipps-checkout-how-it-works-woocommerce.md:39]:
```suggestion
Standard WooCommerce shipping selection is not optimal, so we have integrated with the major Norwegian shipping providers to remove friction from this important step in the sales process. This list currently includes:
```

File: vipps-checkout-how-it-works-woocommerce.md [hunk 5]
```diff
@@ -9,6 +9,57 @@ END_METADATA -->
...
+
+- Bring/Posten
+- Postnord
+- Porterbuddy
+- Helthjem
+- Instabox
+
+If you have agreements with one or more of these providers, you can add their API keys to Vipps Checkout, and we will present a tailored shipping selection window to your customers, giving you this
```
Comment (rebekaburnett) [inline vipps-checkout-how-it-works-woocommerce.md:47]:
```suggestion
If you have agreements with one or more of these providers, you can add their API keys to Vipps Checkout. We will then present a tailored shipping selection window for your customers.
```

File: vipps-checkout-how-it-works-woocommerce.md [hunk 6]
```diff
@@ -9,6 +9,57 @@ END_METADATA -->
...
+
+## Improved shipping selection with Posten
+
+![Posten shipping selection WooCommerce](https://user-images.githubusercontent.com/25223283/226342343-479fb87c-6f4c-4557-8b77-bafd6c36eac7.gif)
+
+instead of this
```
Comment (rebekaburnett) [inline vipps-checkout-how-it-works-woocommerce.md:53]:
```suggestion

```

File: vipps-checkout-how-it-works-woocommerce.md [hunk 7]
```diff
@@ -9,6 +9,57 @@ END_METADATA -->
 
 With [Vipps' official WooCommerce plugin](https://wordpress.org/plugins/woo-vipps/), it's easy to start accepting payments with Vipps, VISA or MasterCard through Vipps Checkout.
 
+<iframe width="100%" height="500" src="https://www.youtube.com/embed/86RSKuQ5GME" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
```
Comment (Haldormyre) [inline vipps-checkout-how-it-works-woocommerce.md:12]: After looking at it i think we should enable the “Privacy enhanced mode” which only sends data to youtube when someone actually clicks the video. (basically change the url from https://www.youtube.com/embed/86RSKuQ5GME to https://www.youtube-nocookie.com/embed/86RSKuQ5GME)
```suggestion
<iframe width="100%" height="500" src="https://www.youtube-nocookie.com/embed/86RSKuQ5GME" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share; fullscreen"></iframe
```

Comment (rebekaburnett) [inline vipps-checkout-how-it-works-woocommerce.md:12]: We can surround it with

File: vipps-checkout-how-it-works-woocommerce.md [hunk 8]
```diff
@@ -9,6 +9,65 @@ END_METADATA -->
...
+
+## How to guide
+
+There are a few steps required to enable the improved shipping selection. For details, we have a [technical how-to-guide](https://vippsas.github.io/vipps-developer-docs/docs/APIs/checkout-api/vipps-checkout-how-it-works-shipping).
+
+We have made a simple video, showing how to enable PorterBuddy in Checkout, but the same principles apply for the other options.
+
+<!-- START_HIDDEN_IN_GITHUB
+
+[Link to video](https://www.youtube.com/watch?v=f4NVq4A73UA)
```
Comment (Haldormyre) [inline vipps-checkout-how-it-works-woocommerce.md:71]:
```suggestion
<iframe width="100%" height="500" src="https://www.youtube-nocookie.com/watch?v=f4NVq4A73UA" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share; fullscreen"></iframe>
```

File: vipps-checkout-how-it-works-woocommerce.md [hunk 9]
```diff
@@ -9,6 +9,65 @@ END_METADATA -->
...
+
+## Standard shipping selection
+
+![Standard shipping selection WooCommerce](https://user-images.githubusercontent.com/25223283/226344800-09395fc7-b1d8-4db3-8815-1d3a71e0a9ab.gif)
+
+## How to guide
+
+There are a few steps required to enable the improved shipping selection. For details, we have a [technical how-to-guide](https://vippsas.github.io/vipps-developer-docs/docs/APIs/checkout-api/vipps-checkout-how-it-works-shipping).
+
+We have made a simple video, showing how to enable PorterBuddy in Checkout, but the same principles apply for the other options.
```
Comment (rebekaburnett) [inline vipps-checkout-how-it-works-woocommerce.md:67]:
```suggestion
We have made a [simple video)(https://www.youtube-nocookie.com/watch?v=f4NVq4A73UA), showing how to enable PorterBuddy in Checkout, but the same principles apply for the other options.
```

**Review by Haldormyre — APPROVED** [review]

**Review by Haldormyre — APPROVED** [review]

**Pull request merged**